testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -460,11 +474,17 @@ def _check_atomgroup(self, ag, u):
 
     def test_atomgroup_guess_bonds(self):
         """Test an atomgroup doing guess bonds"""
-        u = mda.Universe(two_water_gro)
-
-        ag = u.atoms[:3]
-        ag.guess_bonds()
-        self._check_atomgroup(ag, u)
+        def multi_parameters_atomgroup(ff=0.55, lb=0.1, nbonds=2):
+            u = mda.Universe(two_water_gro)
+            ag = u.atoms[:3]
+            ag.guess_bonds(fudge_factor=ff, lower_bound=lb)
+            assert_equal(len(ag.bonds), nbonds)
```

Comment:
in addition to @ianmkenney's comment about using some kind of pytest fixture to do this (probably pytest.parametrize), please don't use `assert_equal` for comparing single ints. Just use a simple `assert`, `assert_equal` for single items is about 10x slower.